tests/src/main/java/com/regnosys/drr/analytics/DefaultValidationSummaryProcessor.java

```diff
@@ -0,0 +1,241 @@
+package com.regnosys.drr.analytics;
```

Comment:
This looks like a lot of code. If this is common - we should find a way to centralise this.